tests_e2e/tests/scripts/agent_cgroups-check_cgroups_agent.py

```diff
@@ -71,6 +71,7 @@ def is_agent_cgroup_controllers_path_exist():
         missing_agent_cgroup_controllers_path.clear()
         verified_agent_cgroup_controllers_path.clear()
         for agenet_cgroup in agent_cgroup_mount_path:
+            all_controllers_path_exist = True
```

Comment:
we verify if agent mounted either in system.slice or azure.slice. On each check, we should reset the default